Libraries/TestHelpers.psm1

```diff
@@ -721,6 +721,40 @@ Function Get-LISAv2Tools($XMLSecretFile) {
 	}
 }
 
+Function Get-SSHKey ($XMLSecretFile) {
+	# Download SSHKey when provide a URL
```

Comment:
is it the only supported way to specify the privatesshkey from the secret file?
Does it support to use a local ssh key file?

Reply:
yes, after update, it only support when specify privatesshkey from the secret file.
yes, it supports local ssh key file.